frontend/stylesheets/dark.less

```diff
@@ -1,6 +1,10 @@
 @import "../../node_modules/antd/dist/antd.dark.less";
 @import "./main.less";
 
+body {
+  background-color: #000;
```

Comment:
To quote from the Refactoring UI pdf:

> True black tends to look pretty unnatural, so start with a *really* dark grey and work your way up to white in steady increments.

I think the same goes for a white background. Maybe you feel like toning these down a tiny notch.

Reply:
This just fixes an inconsistency (see below). Tuning the background is a bigger job. 
<img width="1621" alt="Screenshot 2023-01-13 at 12 00 37" src="https://user-images.githubusercontent.com/335438/212304808-7088db0c-0b63-48d0-bbb9-2d042dca4269.png">